doc/source/cluster/cloud.rst

```diff
@@ -46,6 +46,8 @@ Ray with cloud providers
         $ ray down ray/python/ray/autoscaler/aws/example-full.yaml
 
 
+    AWS Node Provider Maintainer: pdames, Zyiqin-Miranda, DmitriGekhtman, wuisawesome
```

Comment:
Should indicate that these are GitHub handles, like
"AWS Node Provider Maintainer GitHub handles"
or
"AWS Node Provider Maintainers (GitHub handles)"
or sth

Reply:
good idea